website/docs/automation/controlplanes/ack/provision-resources.md

```diff
@@ -1,70 +1,93 @@
+- Create an IAM Policy with necessary DynamoDB permissions
+- Create an IAM Role in AWS with the above permissions
+- Map the Service Account to use the IAM role using Annotations in the Service Account definition.
+
+Fortunately, we have a handy one-liner to help with this process. Run the below:
+
+```bash
+
+$ eksctl create iamserviceaccount --name carts-ack \ 
+  --namespace carts --cluster $EKS_CLUSTER_NAME \
+  --role-name carts-dynamodb-role \
+  --attach-policy-arn $DYNAMODB_POLICY_ARN --approve       
 
-```bash hook=create-secret
-$ kubectl create secret generic catalog-rds-pw \
-  --from-literal=password="$(date +%s | sha256sum | base64 | head -c 32)" -n catalog
-secret/catalog-rds-pw created
 ```
+```eksctl``` provisions a CloudFormation stack to help manage these resources which can be seen in this output
+
+![IRSA screenshot](./assets/eksctl-irsa-cfn.png)
+
+In the AWS Console, navigate to ```IAM -> Roles -> carts-dynamodb-role ``` and the Trust Relationships tab:
 
-Now let's explore the various ACK resources that we'll create. The first is an EC2 security group that will be applied to control access to the RDS database, which is done with a `SecurityGroup` resource:
+![Trusted Entities](./assets/trust-entities.png)
+
+---
+
+
+Now, let's explore how we'll create the DynamoDB Table via a Kubernetes manifest
 
 ```file
-manifests/modules/automation/controlplanes/ack/rds/k8s/rds-security-group.yaml
+manifests/modules/automation/controlplanes/ack/dynamodb/dynamodb-create.yaml
 ```
 
 :::info
 
-The EC2 security group above allows any traffic from the CIDR range of the VPC used by the EKS cluster. This has been done to keep the example clear and understandable. A more secure approach would be to use [Security Groups for Pods](../../../networking/security-groups-for-pods/index.md) to allow traffic from specific pods.
+Astute readers will notice the YAML Spec to be similar to the API endpoints and calls for DynamoDB such as ```tableName``` and ```attributeDefinitions```.
 
 :::
 
-Next we want the RDS database to use the private subnets in our VPC. To accomplish this, we'll create a `DBSubnetGroup` which selects the appropriate subnet IDs:
+Next, we will need to update the regional endpoint for DynamoDB within the Configmap used by the Kustomization to update the **Carts** deployment.
 
 ```file
-manifests/modules/automation/controlplanes/ack/rds/k8s/rds-dbgroup.yaml
+manifests/modules/automation/controlplanes/ack/dynamodb/dynamodb-ack-configmap.yaml
 ```
 
-Finally, we can create the configuration for the RDS database itself with a `DBInstance` resource:
+Next we want to populate the environment variable within the manifest. Run
 
-```file
-manifests/modules/automation/controlplanes/ack/rds/k8s/rds-instance.yaml
-```
+```bash
 
-Apply this configuration to the Amazon EKS cluster:
+$ envsubst < ~/environment/eks-workshop/modules/automation/controlplanes/ack/dynamodb/dynamodb-ack-configmap.yaml \
```

Comment:
Appreciate that you moved away from the kustomize trickery we used before.

To be consistent with the upcoming 1.27 changes can you change this to a single command like:

```
kubectl kustomize ~/environment/eks-workshop/modules/automation/controlplanes/ack/dynamodb \
  | envsubst | kubectl apply -f-
```

Reply:
Done!